UnitTestFrameworkPkg/GoogleTest.md

```diff
@@ -872,6 +920,57 @@ extern "C" {
 }
 ```
 
+#### FunctionMockLib Mocks - Definition for non-Libraries
+The definitions of mock functions for non-libraries, or interfaces that don't have an

```

Comment:
I think `mock functions for non-libraries` can be replaced with `mocks structures that contain function pointers`.

This language covers Protocols, PPIs, and all Service Tables.

Remove use of term `non-libraries' throughout.